test/deploy/linux/kafka/install/roles/prepare/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
The /prepare step is executed very early in the deployer lifecycle. Given the statements seem to be starting something, it might be better to move this role in a /start step instead of /prepare

Reply:
@Julien4218  will update